### DISCUSSION TITLE
Post your queries regarding contribution here

### DISCUSSION THREAD

**Discussion by Abhijit2505** [edualgo/eduAlgo#148] [Q&A] [answered]
Feel free to post your queries here

**Comment by Manoj-Paramsetti** [comment]
Hey @Abhijit2505, 
 Whether the word `hint` you're mentioning is giving an explanation through the function. right?

**Reply by Abhijit2505** [reply]
Yes @Manoj-Paramsetti , the hint is a special function designed for each algorithm that's included in the package.

We have other options to embed documentations in python, but we intentionally use hint function created manually.

**Reply by Abhijit2505** [reply]
> Yes @Manoj-Paramsetti , the hint is a special function designed for each algorithm that's included in the package.
> 
> We have other options to embed documentations in python, but we intentionally use hint function created manually.

This is the answer

**Comment by Abhijit2505** [comment, marked as answer]
I am closing this thread, will open a new one.
